src/MountedTraversal.js

```diff
@@ -50,6 +50,9 @@ export function instHasClassName(inst, className) {
     return false;
   }
   let classes = findDOMNode(inst).className || '';
+  if (typeof classes !== 'string') {
+    return false;
```

Comment:
I have committed your suggested code @Aweary. Kept the string operations as it stands, turns out ```Array.includes``` isn't supported by IE/Edge